css/styles.css

```diff
@@ -0,0 +1,52 @@
+body 	{
+	font-family: cursive;
+	font-size: 18px;
+	margin-top: 0px;
+	margin-right: 0px;
+	background-color: #FCDEBC;
+}
+
+nav	{
```

Comment:
:hammer: I notice that some of your statements have different spaces between the selector and the `{`. Keep it to a single space like so:
```css
img {
  ...
}
```
as opposed to this:
```css
img     {
  ...
}
```